uint8 role
uint32 id
float32 dis
float32 angle0
float32 angle1
float32 fp_rssi
float32 rx_rssi
